topics/cluster-spec.md

```diff
@@ -390,7 +390,7 @@ be issued it can compute the hash slot of the target key and have a
 greater chance of choosing the right node.
 
 An alternative is to just refresh the whole client-side cluster layout
-using the `CLUSTER SLOTS` commands
+using the `CLUSTER SHARDS`, or the deprecated `CLUSTER SLOTS`, command
```

Comment:
If we are deprecating it, shouldn't that be mentioned on the `CLUSTER SLOTS` page with a link to `CLUSTER SHARDS`?

Reply:
Update commands.c to mark `CLUSTER SLOTS` deprecated as well?

Also what about `CLUSTER NODES`? Should it be deprecated too?

Reply:
`CLUSTER NODES` is not deprecated. The redis PR includes a deprecation node of `CLUSTER SLOTS`. It looks like we need to manually deprecate commands, so I can add that here.

Reply:
I guess deprecation notes happen automatically. I did add a note in cluster shards why to use this command instead of `CLUSTER SLOTS`.

Reply:
When CLUSTER SLOTS is flagged as deprecated, it's good to point users to CLUSTER SHARDS from there. Otherwise, users may not find out as easily that CLUSTER SHARDS is its replacement. That's what I meant.

Reply:
What I was saying is that I think there is code that automatically puts a link at the top that points to `CLUSTER SHARDS`. I was looking at `CLUSTER SLAVES`, which has this: https://redis.io/commands/cluster-slaves.

Reply:
Gotcha